geotrek/feedback/forms.py

```diff
@@ -15,8 +41,46 @@ class Meta:
             "problem_magnitude",
             "related_trek",
             "status",
-            "locked",
-            "uid",
-            "origin"
+            "assigned_user"
         ]
         model = Report
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["geom"].required = True
+        if settings.SURICATE_MANAGEMENT_ENABLED:
+            if self.instance.pk:
+                # Store current status
+                self.old_status_id = self.instance.status.suricate_id
+                # Hide fields that are handled automatically in Management mode
+                self.fields["geom"].widget = HiddenInput()
+                self.fields["email"].widget = HiddenInput()
+                self.fields["comment"].widget = HiddenInput()
+                self.fields["activity"].widget = HiddenInput()
+                self.fields["category"].widget = HiddenInput()
+                self.fields["problem_magnitude"].widget = HiddenInput()
+                # Add fields that are used in Management mode
+                # status
+                next_statuses = SURICATE_MANAGEMENT_WORKFLOW[self.instance.status.suricate_id]
+                self.fields["status"].empty_label = None
+                self.fields["status"].queryset = ReportStatus.objects.filter(suricate_id__in=next_statuses)
+                # assigned_user
+                self.fields["assigned_user"].queryset = SelectableUser.objects.filter(userprofile__isnull=False)
+                # message
+                self.fields["message"] = CharField(required=False)
```

Comment:
```suggestion
                self.fields["message"].required = False
```

Reply:
This a new field I am adding, it's not on the model, I have to instanciate it